connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerTest.java

```diff
@@ -1468,36 +1287,7 @@ private void expectTaskHeaderConverter(ClassLoaderUsage classLoaderUsage, Header
         return props;
     }
 
-    private void expectClusterId() {
-        PowerMock.mockStaticPartial(ConnectUtils.class, "lookupKafkaClusterId");
-        EasyMock.expect(ConnectUtils.lookupKafkaClusterId(EasyMock.anyObject())).andReturn("test-cluster").anyTimes();
-    }
 
-    private void expectNewWorkerTask() throws Exception {
-        PowerMock.expectNew(
-                WorkerSourceTask.class, EasyMock.eq(TASK_ID),
-                EasyMock.eq(task),
-                anyObject(TaskStatus.Listener.class),
-                EasyMock.eq(TargetState.STARTED),
-                anyObject(JsonConverter.class),
-                anyObject(JsonConverter.class),
-                anyObject(JsonConverter.class),
-                EasyMock.eq(new TransformationChain<>(Collections.emptyList(), NOOP_OPERATOR)),
-                anyObject(KafkaProducer.class),
-                anyObject(TopicAdmin.class),
-                EasyMock.<Map<String, TopicCreationGroup>>anyObject(),
-                anyObject(OffsetStorageReader.class),
-                anyObject(OffsetStorageWriter.class),
-                EasyMock.eq(config),
-                anyObject(ClusterConfigState.class),
-                anyObject(ConnectMetrics.class),
-                EasyMock.eq(pluginLoader),
-                anyObject(Time.class),
-                anyObject(RetryWithToleranceOperator.class),
-                anyObject(StatusBackingStore.class),
-                anyObject(Executor.class))
-                .andReturn(workerTask);
-    }
     /* Name here needs to be unique as we are testing the aliasing mechanism */
     public static class WorkerTestConnector extends SourceConnector {
```

Comment:
In trunk we now have sample plugins (`SampleSourceConnector`, `SampleSinkConnector`, etc) in this folder. Can we reuse them and get rid of some of these inner classes?

Reply:
I was able to remove the WorkerTestConnector, the rest look to have no replacements currently.

Reply:
It looks like we can also replace `TestConfigurableConverter` and `TestConverter` by `SampleConverterWithHeaders` and all tests still pass.

Reply:
Nice :)